dpdata/plugins/abacus.py

```diff
@@ -1,7 +1,47 @@
+from ast import For
 import dpdata.abacus.scf
 import dpdata.abacus.md
 from dpdata.format import Format
 
+@Format.register("abacus/stru")
+@Format.register("stru")
+class AbacusSTRUFormat(Format):
+    def from_system(self, file_name, **kwargs):
+        return dpdata.abacus.scf.get_frame_from_stru(file_name)
```

Comment:
a blank line should be after this line

Reply:
Got that. I have already added the blank line.